commit e10f2db5d10300c16fa482a136ed31c1aa6e8e8d
Author: Jia Tan <jiat0218@gmail.com>
Date:   Mon Jun 19 23:11:41 2023 +0800

    xz: Slight reword in xz man page for consistency.
    
    Changed will print => prints in xz --robot --version description to
    match --robot --info-memory description.

diff --git a/src/xz/xz.1 b/src/xz/xz.1
index 875b4ff3..1b49bfdb 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -2395,7 +2395,7 @@ may have more columns, but never more than a single line.
 .
 .SS Version
 .B "xz \-\-robot \-\-version"
-will print the version number of
+prints the version number of
 .B xz
 and liblzma in the following format:
 .PP